### PR TITLE
Client authentication/registration using users table

### PR DESCRIPTION
**Pull request by lehone-hp**
Hello @muarachmann this commit is for the clients authentication. 

- I used lara auth and on the users table since for me it is easy and laravel already does all the heavy lifting. 
- I deleted the clients table and used the users table for this. So we will use the admins table for admin and users table for clients. 
- Also added the columns on the users table.

### CODE REVIEW

**Comment by muarachmann** [comment]
Okay @lehone-hp.  Any by you will review in a bit prior to merging

**Comment by lehone-hp** [comment]
> Okay @lehone-hp. Any by you will review in a bit prior to merging

I don't get it!

File: database/migrations/2014_10_12_000000_create_users_table.php [hunk 1]
```diff
@@ -15,9 +15,17 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->increments('id');
-            $table->string('name');
+            $table->string('firstname');
+            $table->string('lastname');
             $table->string('email')->unique();
-            $table->timestamp('email_verified_at')->nullable();
```
Comment (muarachmann) [inline database/migrations/2014_10_12_000000_create_users_table.php]: Is this for like activating the account?

Comment (muarachmann) [reply database/migrations/2014_10_12_000000_create_users_table.php]: You can place also like is status or is active which can be a boolean we send a verification link to the user and on click it activates his/or her account

File: resources/views/auth/register.blade.php [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+                                        <option value="6">Albania</option>
+                                        <option value="7">Bahrain</option>
+                                        <option value="8">Colombia</option>
+                                        <option value="9">Dominican Republic</option>
+                                    </select>
+
+                                    @if ($errors->has('country'))
+                                        <span class="invalid-feedback" role="alert">
+                                            <strong>{{ $errors->first('country') }}</strong>
+                                        </span>
```
Comment (muarachmann) [inline resources/views/auth/register.blade.php]: Overall looks ok except for the issue mentioned above. Testing this in a bit

**Comment by muarachmann** [comment]
Nope not like doing the review before it gets accepted rather testing it so that everything works ok b4 we merge i think so

File: database/migrations/2014_10_12_000000_create_users_table.php (hunk 1 above)
Comment (lehone-hp) [reply database/migrations/2014_10_12_000000_create_users_table.php]: I have added an "isactive" column on the users table

File: resources/views/auth/register.blade.php (hunk 2 above)
Comment (lehone-hp) [reply resources/views/auth/register.blade.php]: ok

**Comment by muarachmann** [comment]
okay then i think it is good to go 👍

**Comment by muarachmann** [comment]
Will be once i get you data codes i will create sample users to simulate the clients interface on the backend and then we can handle products/category stuffs

**Comment by lehone-hp** [comment]
Don't you have like permission to merge PRs or do I have to assign them to you?

**Comment by muarachmann** [comment]
You are the owner so you have to assign me. Its fine at least we going through the codes step by step so you do the merging. I am okay with it

**Comment by lehone-hp** [comment]
> Will be once i get you data codes i will create sample users to simulate the clients interface on the backend and then we can handle products/category stuffs
Okay but for now its just login/register/logout that's working.

**Comment by lehone-hp** [comment]
> You are the owner so you have to assign me. Its fine at least we going through the codes step by step so you do the merging. I am okay with it

okay

**Pull request merged**